npm/cypress-schematic/src/schematics/utils/index.ts

```diff
@@ -46,3 +61,47 @@ export function getLatestNodeVersion (packageName: string): Promise<NodePackage>
     return { name, version }
   }
 }
+
+function generateCTSpec ({ tree, appPath, component }: { tree: Tree, appPath: string, component: any}): void {
+  const buffer = tree.read(`${appPath}/${component['name']}`)
+  const componentString = buffer?.toString()
+  const componentName = componentString?.match(/(?<=class )\S+/g)
+  const componentFilename = component['name'].split('.')[0]
+
+  if (!tree.exists(`${appPath}/${componentFilename}.component.cy.ts`)) {
+    console.log(`Creating component test for: ${componentName ? componentName[0] : componentFilename}`)
+
+    return tree.create(`${appPath}/${componentFilename}.component.cy.ts`, ctSpecContent({ component: componentName ? componentName[0] : componentFilename, file: componentFilename }))
+  }
+}
+
+export function getDirectoriesAndCreateSpecs ({ appPath, tree }: { appPath: string, tree: Tree}) {
+  let components = []
+  let directories = []
+
+  const projectPath = resolve(getSystemPath(normalize('')))
```

Comment:
I believe Angular normalizes all file-paths behind the scene so you should only have to use posix paths

Reply:
The issue is that `getSystemPath` is looking for a `Path` not a `string`.  I'm using `normalize` to convert the string to that Path type.